packages/preset-mini/src/_rules/static.ts

```diff
@@ -70,6 +70,7 @@ export const contentVisibility: Rule[] = [
 
 export const contents: Rule[] = [
   [/^content-\[(.+)\]$/, ([, v]) => ({ content: `"${v}"` })],
+  [/^content-raw-\[(.+)\]$/, ([, v]) => ({ content: `"${v}"` })],
```

Comment:
```suggestion
  [/^content-raw-\[(.+)\]$/, ([, v]) => ({ content: v })],
```

Isn't this supported to be like this?